src/commands/createStaticWebApp/createStaticWebApp.ts

```diff
@@ -52,6 +51,7 @@ export async function createStaticWebApp(context: IActionContext & Partial<ICrea
             }
         }
 
+        await promptForDefaultBranch(context, verifiedWorkspace.repo);
```

Comment:
Instead of moving this line, it would be better if you passed `verifiedWorkspace` into `promptForDefaultBranch` and use `workspace.remoteRepo.default_branch` directly instead of referencing `context.branchData.name`. Then the order doesn't matter and plus `remoteRepo.default_branch` is a more intuitive name IMO